middlewares/cookie-parser.js

```diff
@@ -0,0 +1,13 @@
+export default (req, res, next) => {
```

Comment:
https://www.npmjs.com/package/cookie-parser

Reply:
yea, I know, but according to the task I thought that we need to implement our own parser